### ISSUE TITLE
LAB 5 관련 질문

### ISSUE THREAD

**Issue by dkwanm1** [swsnu/swppfall2019#104]
![image](https://user-images.githubusercontent.com/48665265/66265969-ac2eea80-e859-11e9-98e6-96fd015340ac.png)
seupTest.js 파일 안에서 이 부분을 작성하면
저희가 직접 componentDidMount() 안에 작성하는 부분들은 스킵하고 테스팅하는건가요?

**Comment by kyunggeun-lee** [comment]
네 맞습니다. 저 옵션을 true로 설정하면 컴포넌트가 마운트된 후 componentDidMount를 호출하지 않고, setProps나 setContext 이후에 componentDidUpdate 를 호출하지 않습니다.
참조: https://airbnb.io/enzyme/docs/api/ShallowWrapper/shallow.html

**하지만 shallow() 로 렌더링했을때에만 해당됩니다.** mount로 렌더링하는 경우에는 언제나 componentDidMount와 componenetDidUpdate가 호출됩니다.
